app/helpers/dashboards_helper.rb

```diff
@@ -1,7 +1,7 @@
 # Convenience methods consumed in the dashboards controller index view
 module DashboardsHelper
   def date_range(date: Time.zone.now)
-    if (Config.start_day.downcase == :monthly )
+    if (Config.start_day == :monthly )
```

Comment:
just to check, this is because the downcasing is handled in the `Config.start_day` class method?

Reply:
Correct!